### ISSUE TITLE
Julie Bishop's speech incorrect

### ISSUE THREAD

**Issue by henare** [drzax/morph-mp-first-speech#1]
It's pointing [to one](http://parlinfo.aph.gov.au/parlInfo/search/display/display.w3p;query=Id%3A%22chamber%2Fhansardr%2F1998-11-23%2F0008%22) by Joel Fitzgibbon.

**Comment by drzax** [comment]
Quite correct, and Joel Fitzgibbon's is incorrect too. Here's a full list of the incorrect links I'm aware of which have been reported to the Parliamentary Library.

Joel Fitzgibbon
Kevin Andrews
Bronwyn Bishop
Mal Brough
Glenn Lazarus
Alex Gallacher
Michael Danby
John Madigan
Jill Hall
Alan Griffin
Lee Rhiannon
Eric Hutchison
Larissa Waters
Zhenya Wang
Penny Wright
Andrew Nikolic
Tanya Plibersek
Christopher Pyne
Philip Ruddock
Bernie Ripoll
Bruce Scott
Wayne Swan
Warren Truss
Brett Whiteley
Anthony Albanese

**Comment by henare** [comment]
> ...which have been reported to the Parliamentary Library.

Wow, so it's the upstream data? Welcome to my world :-/

**Comment by drzax** [comment]
Yeah, and to be fair, I think it's the upstream data for them too (or some similar legacy technology issue).
